### PR TITLE
Update issue templates

### LINKED ISSUE #110: Add Issue Template for OpenRegistry
Create issue template to facilitate smooth experience for issue submission.
Similarly, create Feature templates with required fields.
This will help issue and feature submissions descriptive and easier to work on.

### PR DESCRIPTION
**Pull request by guacamole**
this PR fixes #110

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: guacamole
```
Comment (jay-dee7) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: @guacamole lets remove assignees field here

**Review by jay-dee7 — CHANGES_REQUESTED** [review]
just remove assignees from here and all is good

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (guacamole) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done!

**Review by jay-dee7 — APPROVED** [review]
LGTM @guacamole. Just waiting for failing tests to pass

**Pull request merged**